lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -462,25 +461,139 @@ def test_team_deleted(self):
         self._verify_cell_data_for_user(self.student2.username, self.course.id, 'Team Name', team2.name)
 
 
-class TestProblemResponsesReport(TestReportMixin, InstructorTaskCourseTestCase):
+# pylint: disable=protected-access
+class TestProblemResponsesReport(TestReportMixin, InstructorTaskModuleTestCase):
     """
     Tests that generation of CSV files listing student answers to a
     given problem works.
     """
     def setUp(self):
         super(TestProblemResponsesReport, self).setUp()
-        self.course = CourseFactory.create()
+        self.initialize_course()
+        self.instructor = self.create_instructor('instructor')
+        self.student = self.create_student('student')
+
+    # Can be used once CapaDescriptor gets ``generate_report_data`` method.
```

Comment:
This means "once https://github.com/edx/edx-platform/pull/17948 merges", correct?

Reply:
@efischer19  Yes, I tested this PR with #17948, so I left this snippet around to make it easy to recreate those tests.